main.tf

```diff
@@ -89,6 +89,13 @@ resource "aws_wafv2_web_acl" "main" {
         }
       }
 
+      dynamic "rule_label" {
+        for_each = length(lookup(rule.value, "rule_label", {})) == 0 ? [] : [1]
```

Comment:
should be an array `rule_labels` so we are not limited to just 1 rule label, similar to `custom_reponse_bodies` in the previous PR

Reply:
Yeah, you're right. I was just doing this as an example really. Want to just handle in your open PR?